XLS-20/README.md

```diff
@@ -0,0 +1,828 @@
+
+Such objects can also be used to implement non-fungible tokens (NFTokens), as seen in an example implementation by the **XRPL Labs** team.
+
+Non-fungible tokens serve to encode ownership of physical, non-physical or purely digital goods, such as works of art and in-game items.
+
+This proposal introduces extensions to the XRP Ledger that would support a native non-fungible token type, along with operations to enumerate, purchase, sell and hold such tokens.
+
+While other proposals (some of which are _extremely_ interesting) have been made, the authors believe that this proposal represents a strong commitment to supporting NFTs on the XRP Ledger, and adds a rich set of flexible primitives that can be used by token issuers.
+
+The non-fungible tokens proposed are:
+
+- **Indivisible**;
+- **Unique**; and
+- **Not used for payments**.
+
+### 1.1.1. Advantages and Disadvantages
+
+**Advantages**
+
+- NFT-specific configurations and options such as transfer fees and burnable/non-burnable tokens allow for flexibility.
+- An efficient storage mechanism, capable of storing a large number of NFTs.
+
+**Disadvantages**
+
+- Requires an amendment to the XRP Ledger protocol, which increases complexity and adds more types of data that must be tracked and maintained as part of the ledger indefinitely.
+- New transaction and data types require new implementation code from client libraries and wallets to read, display, and transact with NFTs.
+
+## 1.2. Creating and Transferring Tokens on XRPL
+
+### 1.2.1. On-Ledger Data Structures
+
+We propose two new objects and one new ledger structure:
+
+1. An **`NFToken`** is a new object that describes a single NFT.
+2. An **`NFTokenOffer`** is a new object that describes an offer to buy or sell a single `NFToken`.
+3. An **`NFTokenPage`** is a ledger structure that contains a set of **`NFToken`** objects owned by the same account.
+
+#### 1.2.1.1. The **`NFToken`** object
+
+The **`NFToken`** object represents a single NFT and holds data associated with the NFT itself. NFTs are created using the **`NFTokenMint`** transaction and can, optionally, be destroyed by the **`NFTokenBurn`** transaction.
+
+##### 1.2.1.1.1. Fields
+
+An **`NFToken`** object can have the following required and optional fields. Notice that, unlike other objects, no field is needed to identify the object type or current owner of the object, because NFTs are grouped into pages that implicitly define the object type and identify the owner.
+
+---
+
+| Field Name        | Required?        |  JSON Type      | Internal Type     |
+|-------------------|:----------------:|:---------------:|:-----------------:|
+| `NFTokenID`        |:heavy_check_mark:|`string`         | `UINT256`         |
+
+This composite field uniquely identifiers a token; it contains:
+* a set of 16 bits that identify flags or settings specific to the NFT
+* 16 bits that encode the transfer fee associated with this token
+*if
```

Comment:
Lines 70-71
* 16 bits that encode the transfer fee associated with this token, if any